install/installation-archlinux.md

```diff
@@ -0,0 +1,113 @@
+# Install self-hosted TimescaleDB on archlinux-based systems
+You can host TimescaleDB yourself, on your Arch Linux system. These
+instructions use the `pacman` package manager.
+
+<highlight type="warning">
+If you have already installed PostgreSQL using a method other than the `pacman`
+package manager, you could encounter errors following these instructions. It is
+safest to remove any existing PostgreSQL installations before you begin. If you
+want to keep your current PostgreSQL installation, do not install TimescaleDB
+using this method.
+[Install from source](/install/latest/self-hosted/installation-source/)
+instead.
+</highlight>
+
+<procedure>
+
+### Installing self-hosted TimescaleDB on archlinux-based systems
+1.  Install TimescaleDB and timescaledb-tune:
+    ```bash
+    pacman -Syu timescaledb timescaledb-tune
+    ```
+1.  Initialize the database as the postgres user: 
+    ```bash
+    sudo -u postgres initdb --locale=en_US.UTF-8 --encoding=UTF8 -D /var/lib/postgres/data --data-checksums
+    ```
+1.  Run timescaledb-tune to adjust your `postgresql.conf` file, to use TimescaleDB as PostgreSQL extension:
+    ```bash
+    timescaledb-tune
+    ```
+1. Enable and start the service:
+    ```
+    systemctl enable postgresql.service
+    systemctl start postgresql.service
+    ```
+1.  Restart the service:
+    ```bash
+    systemctl restart postgresql
+    ```
```

Comment:
This step is redundant. You started the database in previous step and did no changes that require restart
```suggestion
```